src/llfs/data_packer.hpp

```diff
@@ -316,7 +316,6 @@ class DataPacker
     }
 
     // This will set the `which` field of the variant record.
-    //
```

Comment:
nit: Please revert!  (I like the `//` connecting lines ;-))

Reply:
done :)